README.md

```diff
@@ -67,36 +67,54 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
      cd orchestrator-helm-chart/charts
      ```
 
-2. Create a namespace for the Orchestrator solution:
+2. Deploy the PostgreSQL reference implementation for persistence support in SonataFlow following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/postgresql/README.md)
+
+3. Create a namespace for the Orchestrator solution:
 
    ```console
    oc new-project orchestrator
    ```
 
-3. Set GITHUB_TOKEN environment variable
+4. Set `GITHUB_TOKEN` environment variable
 
    ```console
    export GITHUB_TOKEN=<github token>
    ```
 
-4. Deploy PostgreSQL reference implementation following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/postgresql/README.md)
+5. Create a namespace for the Red Hat Developer Hub Operator (RHDH Operator):
 
-### ...without GitOps
+   ```console
+   oc new-project rhdh-operator
+   ```
 
-1.  Install the orchestrator Helm chart:
+6. Generate a random alphanumeric string of 24 characters to be used in the `BACKEND_SECRET`:
 
-    ```console
-    helm upgrade -i orchestrator orchestrator --set rhdhOperator.github.token=$GITHUB_TOKEN
-    ```
+   ```console
+   ewxport BACKEND_SECRET=$(mktemp -u XXXXXXXXXXXXXXXXXXXXXXX)
+   ```
+
+###  Deployment without GitOps
+
+7. Create the secret in the 'rhdh-operator' namespace with the GitHub token:
+
+   ```console
+   oc create secret generic backstage-backend-auth-secret --from-literal=BACKEND_SECRET=$BACKEND_SECRET --from-literal=GITHUB_TOKEN=$GITHUB_TOKEN
+   ```
+
+   Install the orchestrator Helm chart:
+
+   ```console
+   helm upgrade -i orchestrator orchestrator
```

Comment:
before starting the installation we should switch from `rhdh-operator` project/namespace to `orchestrator` or  change the installation command to specify the namespace

Reply:
Done